### PR TITLE
Update Function.cs

### PR DESCRIPTION
**Pull request by rajputvishwas**
Update function.cs  as per requirement

### CODE REVIEW

File: Function.cs [hunk 1]
```diff
@@ -17,6 +17,14 @@ public class Function
 
         private readonly string[] coolColors =
         {
+			"Air Force Blue",
+			"American rose",
+			"Anroid Green",
```
Comment (javajudt) [inline Function.cs]: Looks like you're just missing the "d" in "Android".

```suggestion
			"Android Green",
```

**Review by javajudt — CHANGES_REQUESTED** [review]
Great list! Just a minor typo to fix.

**Comment by rajputvishwas** [comment]
Updated ☺

**Review by javajudt — APPROVED** [review]
Perfect! 👌

**Pull request merged**